pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
```

Comment:
To switch to flit you'll need to enable tox isolated_build and delete the setup.py:

https://tox.readthedocs.io/en/latest/config.html

Reply:
You may prefer to use:

```
pipx run setup-py-upgrade .
```

and then use the setuptools build-system